### ISSUE TITLE
The converted keras model is too slow in react.js node

### ISSUE THREAD

**Issue by amarnathresearch** [tensorflow/tfjs#5181]
I have created a keras model for detection in images. The model gives good accuracy with a latency time of 0.06 seconds (25+ Frames Per Second) on average when tested in CPU system (python3.6). I followed the steps in the documentation (https://github.com/tensorflow/tfjs-converter/tree/master/tfjs-converter) to convert the Keras model to tensorflowjs. The conversion is successful and loaded into the node react as shown below

`model.current = await tf.setBackend('wasm').then(() =>tf.loadLayersModel('http:/model/model.js'));
`
The model is loaded successfully, but the inference time is bad as 0.8 seconds (1 frame per second) on average.

The conversion code is given below:

`# !tensorflowjs_converter --input_format keras  --quantize_uint8=* /content/drive/MyDrive/model/model.h5 /content/drive/MyDrive/detect/ `

alternate option is

`!tensorflowjs_converter --input_format=keras --quantize_uint8=* --strip_debug_ops=True --output_node_names='Postprocessor/ExpandDims_1,Postprocessor/Slice' /content/drive/MyDrive/model/model.h5 /content/drive/MyDrive/detect/ `

Any suggestion, please.

**Comment by rthadur** [comment]
As mentioned here https://github.com/tensorflow/tfjs/issues/4431#issuecomment-751865544 , wasm is not supoorted in react native.
